gulp/tasks/test.js

```diff
@@ -95,6 +95,9 @@ function _getProtractorParams() {
   // If --version=number flag is added on the command-line.
   params = _addCommandLineFlag( params, commandLineParams, 'version' );
 
+  // If --suite=suite1,suite2 flag is added on the command-line.
+  params = _addCommandLineFlag( params, commandLineParams, 'suite' );
+
   return params;
```

Comment:
I agree that this code isn't the clearest; I left it as is just to keep the set of changes in this PR small.